integration_tests/tests/in_cluster.rs

```diff
@@ -118,68 +125,84 @@ impl InClusterJanusPair {
+                api_url: Self::in_cluster_aggregator_api_url(&leader_namespace),
                 bearer_token: leader_aggregator_api_auth_token,
+                is_first_party: true,
             })
-            .await;
+            .await
+            .unwrap();
 
         let paired_helper_aggregator = divviup_api
-            .pair_aggregator(
-                &account,
-                &NewAggregatorRequest {
+            .create_aggregator(
+                account.id,
+                NewAggregator {
                     name: "helper".to_string(),
-                    api_url: Self::in_cluster_aggregator_api_url(&helper_namespace).to_string(),
+                    api_url: Self::in_cluster_aggregator_api_url(&helper_namespace),
                     bearer_token: helper_aggregator_api_auth_token,
                 },
             )
-            .await;
+            .await
+            .unwrap();
 
         let collector_hpke_config = divviup_api
             .create_hpke_config(
-                &account,
-                &NewHpkeConfigRequest {
-                    name: "Integration test key".to_string(),
-                    contents: STANDARD.encode(task.collector_hpke_config().unwrap().get_encoded()),
-                },
+                account.id,
+                STANDARD.encode(task.collector_hpke_config().unwrap().get_encoded()),
+                Some("Integration test key"),
             )
-            .await;
+            .await
+            .unwrap();
 
-        let provision_task_request = NewTaskRequest {
+        let provision_task_request = NewTask {
             name: "Integration test task".to_string(),
             leader_aggregator_id: paired_leader_aggregator.id,
             helper_aggregator_id: paired_helper_aggregator.id,
-            vdaf: task.vdaf().try_into().unwrap(),
+            vdaf: match task.vdaf().to_owned() {
+                VdafInstance::Prio3Count => Vdaf::Count,
+                VdafInstance::Prio3Sum { bits } => Vdaf::Sum {
+                    bits: bits.try_into().unwrap(),
+                },
+                VdafInstance::Prio3SumVec { bits, length } => Vdaf::SumVec {
+                    bits: bits.try_into().unwrap(),
+                    length: length.try_into().unwrap(),
+                },
+                VdafInstance::Prio3Histogram { buckets } => Vdaf::Histogram { buckets },
+                VdafInstance::Prio3CountVec { length } => Vdaf::CountVec {
+                    length: length.try_into().unwrap(),
+                },
+                other => panic!("unsupported vdaf {other:?}"),
+            },
             min_batch_size: task.min_batch_size(),
             max_batch_size: match task.query_type() {
                 QueryType::TimeInterval => None,
                 QueryType::FixedSize { max_batch_size, .. } => Some(*max_batch_size),
             },
-            expiration: "3000-01-01T00:00:00Z".to_owned(),
+            expiration: None,
```

Comment:
this is the only known functional change here, and i don't know if it'll break anything. it was easier to type "None" than parse an OffsetDateTime